app/subscriber/src/features/filter-media/FilterMediaLanding.tsx

```diff
@@ -0,0 +1,161 @@
+import { PageSection } from 'components/section';
+import { ISubMediaGroupItem } from 'features/search-page/components/advanced-search/interfaces';
+import React from 'react';
+import { useContent, useLookup } from 'store/hooks';
+import { Col, ISourceModel, Row, Show } from 'tno-core';
+
+import { FilterMedia } from './FilterMedia';
+import * as styled from './styled';
+import { alphabetArray } from './utils';
+
+/** The FilterMediaLanding Consists of the filter to narrow down the results of media, as well as the results. It is returned in a two column layout*/
+export const FilterMediaLanding: React.FC = () => {
+  const [
+    {
+      mediaType: { filter },
+    },
+    { storeMediaTypeFilter: storeFilter },
+  ] = useContent();
+  const [{ sources, mediaTypes }] = useLookup();
+  const [activeFilter, setActiveFilter] = React.useState<ISubMediaGroupItem>();
+
+  const [activeLetter, setActiveLetter] = React.useState<string>('All');
+  const [narrowedOptions, setNarrowedOptions] = React.useState<ISourceModel[]>([]);
+  const [activeSource, setActiveSource] = React.useState<ISourceModel | null>(null);
+  const [subMediaGroups, setSubMediaGroups] = React.useState<ISubMediaGroupItem[]>();
+
+  React.useEffect(() => {
```

Comment:
Making another ticket to get rid of this duplication and share it (also used in `MediaSection`